models/spring-ai-openai/src/test/java/org/springframework/ai/openai/chat/MessageTypeContentTests.java

```diff
@@ -44,7 +44,7 @@
 import org.springframework.util.MultiValueMap;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.when;
+import static org.mockito.BDDMockito.given;
```

Comment:
actually I disabled this checkstyle when adding some tests in another PR, I guess I need to learn the bddmockito way.